src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Draw {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public double calculateArea() {
+        return 0;
```

Comment:
The `calculateArea` method returns 0, which might not be meaningful for a base class. Consider making this method abstract to enforce subclasses to provide their own implementation.